src/components/layout/SceneContainer.tsx

```diff
@@ -4,13 +4,15 @@ import { UndoInsetStyle } from '../common/SceneWrapper'
 import { styled } from '../hoc/styled'
 
 interface Props {
+  expand?: boolean
   undoTop?: boolean
   undoRight?: boolean
   undoBottom?: boolean
   undoLeft?: boolean
   undoInsetStyle?: UndoInsetStyle
 }
-export const SceneContainer = styled(View)<Props>(theme => ({ undoTop, undoRight, undoBottom, undoLeft, undoInsetStyle }) => ({
+export const SceneContainer = styled(View)<Props>(theme => ({ expand, undoTop, undoRight, undoBottom, undoLeft, undoInsetStyle }) => ({
```

Comment:
Add a `title` prop and handle the positioning of `SceneHeaderUi4` inside here. This way we don't have to use additional views to layout the scene header along with the rest of the scene contents. `SceneContainer` can then be the sole provider of all the 0.5rem margins around the edges of the scene and no custom spacing will be required for UI4 compatible components.

Reply:
I'm going to push back on this request a bit. Why must SceneContainer extend SceneHeaderUi4? The coupling would cause more work as SceneHeaderUi4 ever changes it's API, unless you're claiming ossification?

Reply:
For business logic, sure, composition all the way. This is a visual concept that applies anywhere there is a SceneHeaderUi4, inheritance is the way to go.

There's never a situation where we want to change the title layout for one scene independent of another, it's a global visual concept. Therefore, there's no reason to give the next developer the freedom to compose.